bootstrap.py

```diff
@@ -5,6 +5,7 @@
 
 Use `pydoc ./bootstrap.py` to get the documentation.
 Use `awk -F'# >' 'NF>1 {print $2}' ./bootstrap.py` to see the flow.
+Use `/usr/libexec/platform-python bootstrap.py` on RHEL8
```

Comment:
Is that important here?

Reply:
no

Reply:
I think you can just merge and loose that or you prefer I force push to remove it.?

Reply:
Nah, I'm good.